org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -160,17 +171,20 @@ def registerDeviceChange() {
         if(subscriptionEndpt != null){
 			if(state.deviceSubscriptionMap[deviceId] == null){
 				state.deviceSubscriptionMap.put(deviceId, [subscriptionEndpt])
-				log.info "Added subscription URL: ${subscriptionEndpt} for ${myDevice.displayName}"
```

Comment:
This is the last bit of feedback, looks good otherwise!

---
In reply to: [112693668](https://github.com/openT2T/translators/pull/234#discussion_r112693668) [](ancestors = 112693668)